server/src/main/kotlin/org/javacs/kt/index/SymbolIndex.kt

```diff
@@ -110,7 +110,7 @@ class SymbolIndex(
                     addDeclarations(allDescriptors(module, exclusions))
 
                     val finished = System.currentTimeMillis()
-                    val count = Symbols.slice(Symbols.fqName.count()).selectAll().first()[Symbols.fqName.count()]
+                    val count = Symbols.selectAll().first()[Symbols.fqName.count()]
```

Comment:
this is the changed of exposed, please review it, I do not know if it is the same of origin logic